src/lib_json/json_writer.cpp

```diff
@@ -262,6 +262,14 @@ static String toHex16Bit(unsigned int x) {
   return result;
 }
 
+static void appendRaw(String& result, unsigned ch) {
+  result += static_cast<char>(ch);
+}
+
+static void appendHex(String& result, unsigned ch) {
```

Comment:
Can we use `inline ` method?

Reply:
I think it made a difference 20 years ago, but it makes no difference for a function like this. I try hard to avoid adding keywords that have no effect. You can try it on gcc.godbolt.org or something. The optimizers are amazing.

Reply:
I agree that optimizes are amazing these days, and yet again I will do my best to not rely on the optimizer if that will not make the code looks bad. Using `inline` or `constexpr` would not make the code worse here. Optimizer is good, but is not part of any standard, so you never can be sure what and where it will optimize given tons of different target architectures and several different compilers.